pkgs/development/python-modules/scs/default.nix

```diff
@@ -5,26 +5,22 @@
 , lapack
 , numpy
 , scipy
-, scs
```

Comment:
I removed the scs dependency because the python package uses a different commit for the scs library (as a git submodule) than the official v3.0.0 tag. Otherwise, I would have to re-fetch/override the source. The official scs v3.0.0 tagged-source doesn't work in this package (fails test), needs the submodule's commit.